app/api-requests/eligibility-api.js

```diff
@@ -47,7 +47,28 @@ async function getEligibleBusinesses (businessEmail) {
   }
 }
 
+async function checkWaitingList (businessEmail) {
+  console.log(`${new Date().toISOString()} Checking waiting list for : ${JSON.stringify({ businessEmail })}`)
+  try {
+    const response = await Wreck.get(
+        `${config.eligibilityApi.uri}/waiting-list?emailAddress=${businessEmail}`,
+        { json: true }
+    )
+    if (response.res.statusCode !== 200 && response.res.statusCode !== 302) {
+      throw new Error(`HTTP ${response.res.statusCode} (${response.res.statusMessage})`)
+    }
+    console.log(`${new Date().toISOString()} waiting list API returned: ${JSON.stringify(response.payload)}`)
+    return response.payload
+  } catch (error) {
+    console.error(`${new Date().toISOString()} Request to the waiting list API failed: ${JSON.stringify({
+        businessEmail
+      })}`, error)
+    return []
```

Comment:
i think we cannot return [] here:

const { alreadyRegistered, accessGranted } = await checkWaitingList(request.payload.emailAddress)

Reply:
This is in the catch block so I think will return a default value of false for both booleans (I maybe wrong though)

Reply:
refactored this to not return this empty array